Layout changes to fix DRC errors for CI2404 tapeout

RTimothyEdwards opened the pull request:
Modified .mag files to make corrections for tapeout (mainly corrected MiM cap bottom plate spacing errors, and renamed the "level_shifter" cell to avoid a naming conflict in another project).

RTimothyEdwards committed:
Modified .mag files to make corrections for tapeout (mainly
corrected MiM cap bottom plate spacing errors, and renamed the
"level_shifter" cell to avoid a naming conflict in another
project).

htfab commented:
Thanks, merged.

I don't see the renamed level_shifter in the commit, but it's not needed for the final design anyway. The two parts of it, level_shifter_ad & level_shifter_dd are placed in separate deep nwells, so the combined level_shifter.mag was only used for LVS againt level_shifter.sch.

RTimothyEdwards commented:
Sorry, I thought I renamed the level shifter, but apparently I just removed it because it was unused.  You can put it back (with a different name) if you think it should be in the repository for reference, future use, or whatever.

htfab commented:
No problem, I don't think I will need it and it will be in the repo history anyway. Thanks again for the fixes.
